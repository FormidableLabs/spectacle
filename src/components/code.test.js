import React from 'react';
import { mount } from 'enzyme';
import Code from './code';

describe('<Code />', () => {
  test('should render correctly.', () => {
    const context = { styles: { components: { code: { fontWeight: 500 } } } };
    const wrapper = mount(<Code>const [a, ...b] = [1, 2, 3, 4]</Code>, {
      context
    });
    expect(wrapper).toMatchSnapshot();
  });
});
