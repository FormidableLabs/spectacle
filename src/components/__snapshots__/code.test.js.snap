// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Code /> should render correctly. 1`] = `
<Code>
  <Styled(code)
    className={undefined}
    dangerouslySetInnerHTML={
      Object {
        "__html": "const [a, ...b] = [1, 2, 3, 4]",
      }
    }
    styles={
      Array [
        Object {
          "fontWeight": 500,
        },
        Object {},
        undefined,
      ]
    }
  >
    <code
      className="css-10rvbm3"
      dangerouslySetInnerHTML={
        Object {
          "__html": "const [a, ...b] = [1, 2, 3, 4]",
        }
      }
    />
  </Styled(code)>
</Code>
`;
